parm/config/config.aeroic

```diff
@@ -0,0 +1,14 @@
+#!/bin/ksh -x
+
+########## config.aeroic ##########
+# Append aerosol variables to FV3 initial conditions
+
+echo "BEGIN: config.aeroic"
+
+# Task and thread configuration
+export wtime_aeroic="00:30:00"
+export npe_aeroic=1
+export npe_node_aeroic=3
+export nth_aeroic=8
```

Comment:
All of this needs to be added to a new "aeroic" block in config.resources. Change this config to match the others with regards to sourcing config.resources with "aeroic" as the argument after moving resources to config.resources. See this config as an example of what will remain in this config:

https://github.com/NOAA-EMC/global-workflow/blob/develop/parm/config/config.analcalc